projects/app/js/app.js

```
@@ -1102,15 +1102,27 @@ async function syncState() {
         if (alarmsTab && !alarmsTab.classList.contains('hidden')) {
             await renderBusinessDays();
             // Skip re-rendering alarms list if user is actively interacting with an input/select inside it
-            if (!activeEl || !alarmsTab.contains(activeEl)) {
+            const alarmList = getEl(ID_ALARM_LIST);
+            const businessDaysContainer = getEl(ID_BUSINESS_DAYS_CONTAINER);
+            const isEditing = activeEl && (
+                alarmList?.contains(activeEl) ||
+                businessDaysContainer?.contains(activeEl)
+            );
```

ESLint

[error] Replace `⏎················alarmList?.contains(activeEl)·||⏎················businessDaysContainer?.contains(activeEl)⏎············` with `alarmList?.contains(activeEl)·||·businessDaysContainer?.contains(activeEl)`

(prettier/prettier)